### ISSUE TITLE
A11y_PowershellEditorServices_Terminal_Keyboard : Tooltip is not visible for the keyboard only users.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61697]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10
NVDA 2018.2.1

Additional Details:
Keyboard ,MAS1.1.1 ,MAS1.3.1

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate to "Terminal",  "PowerShell Integrated Console" ,"New terminal" etc controls.
4.Navigate using keyboard & see whether tooltip is visible or not.

**Actual:**
Using mouse tooltip is visible for all the control present in the terminal tab , But for the keyboard only user it is not visible.

**Expected:**
For the keyboard only users tooltip should be visible.

Same issue is observed with NVDA

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to get all the information about the control.

**MAS Reference:**
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Does this issue occur when all extensions are disabled?: Yes
[A11y_PowershellEditorServices_Terminal_Keyboard1.pptx](https://github.com/Microsoft/vscode/files/2510064/A11y_PowershellEditorServices_Terminal_Keyboard1.pptx)

**Comment by Tyriar** [comment]
@isidorn the panel buttons don't seem to have content or `aria-label`s?

<img width="324" alt="screen shot 2018-12-13 at 8 33 59 am" src="https://user-images.githubusercontent.com/2193314/49952942-e9cd1c00-feb1-11e8-8bdb-e4ff31521932.png">

**Comment by isidorn** [comment]
This is a duplicate of https://github.com/Microsoft/vscode/issues/52016
